src/olympia/blocklist/mlbf.py

```diff
@@ -19,6 +19,15 @@
 log = olympia.core.logger.getLogger('z.amo.blocklist')
 
 
+def diff_lists(
+    previous: List[str], current: List[str]
+) -> Tuple[Set[str], Set[str], int]:
+    extras = set(current) - set(previous)
+    deletes = set(previous) - set(current)
+    changed_count = len(extras) + len(deletes) if len(previous) > 0 else len(current)
```

Comment:
```suggestion
    changed_count = len(extras) + len(deletes)
```
the `if` is redundant?

Reply:
Done.